src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -77,20 +77,13 @@ export function IndexPatternProvider(Private, $http, config, kbnIndex, Promise,
 
   function updateFromElasticSearch(indexPattern, response) {
     if (!response.found) {
-      // custom notification provides link to create index pattern with missing id
-      const notificationMessage = `Could not locate that index-pattern (id: ${indexPattern.id})`;
-      const actions = [{
-        text: 'OK',
-        dataTestSubj: 'reportCompleteOkToastButton'
-      }, {
-        text: 'Create',
-        callback: () => {
-          kbnUrl.change('/management/kibana/index?id={{id}}&name=', { id: indexPattern.id });
-        }
-      }];
+      const markdownSaveId = indexPattern.id.replace('*', '%2A');
```

Comment:
@spalger, there is a bug in marked that when a URL contains an asterisk it won't render the URL.

That project is really in a bad place - in the future we might want to find something else: https://github.com/chjj/marked/issues